src/sui-link-item/index.jsx

```diff
@@ -51,7 +53,8 @@ export default class LinkItem extends Component {
           target={target}
           title={title}
           to={url}
-          onClick={handleClick}>
+          onClick={handleClick}
+          rel={nofollow ? 'nofollow' : null}>
```

Comment:
No. I have tested and in case nofollow values is false the component not rendered any rel attribute.